var/spack/repos/builtin/packages/py-pyvcf/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPyvcf(PythonPackage):
+    """PyVCF - A Variant Call Format Parser for Python"""
+
+    homepage = "https://github.com/jamescasbon/PyVCF"
+    url      = "https://github.com/jamescasbon/PyVCF/archive/v0.6.0.tar.gz"
+
+    version('0.6.7', '51b57ce99e0c2f7be2a18d08d8f87734')
+    version('0.6.0', '5e38a610b1bf837e5e74a748e281cec7')
+
```

Comment:
Still missing a dependency on `distribute`.

Reply:
`Distribute` is a now deprecated fork of the `Setuptools` project.

https://pypi.python.org/pypi/distribute/0.7.3
This package is a simple compatibility layer that installs Setuptools 0.7+.

Reply:
I get that, but the `setup.py` lists it as a required dependency, so if you don't make it explicit, then Python will try to download it automatically. This will cause the package to break for users on an air-gapped network.

Reply:
I have built this on a number of occasions and have yet to witness the automatic download/build of ```Distribute```.  I am hesitant...

Reply:
Perhaps you already have distribute installed.

Reply:
I cannot find a python package definition in spack for distribute.  I assumed that it would be something like "py-distribute" or py-pydistribute" but neither of these are in spack; nor anything else that I can construe as an equivalent.  Further, I used the python console to search for installed packages that might have interfered with the spack build dependencies - some basis for the "non-error"...

I am finding it very hard to locate anything that could explain this situation.  Please, please, please... if there is a python expert out there that can point me in the roper direction.  My compass seems to be faulty.

Reply:
It appears `distribute` was added in version 0.4.5 for use_2to3 support and removed after a couple of years in favor of updated `setuptools` starting with version 0.6.6.  Would it be acceptable to just note that only Python 2.x is currently supported for versions <0.6.6?  I'm not sure the challenges of getting `distribute` functioning is worth the effort mentioned in PR #3393.

Reply:
Since this is a new package, we can easily drop 0.6.0 support and only include the latest version, thereby removing the `distribute` dependency. Is there any reason you need to add 0.6.0?

Also, I noticed that this is downloading from a fork of an original project on GitHub, which makes me a little uneasy. Version 0.6.6 does not even exist on this fork, so this package should crash immediately for anyone who tries to install it. Let's download from PyPI instead?